### PR TITLE
Add of Czech translate

### PR DESCRIPTION
**Pull request by redoper1**

### CODE REVIEW

**Comment by redoper1** [comment]
Yeah, I'm an Czech native speaker too. I will fix it as soon as I will return from the work.

**Comment by redoper1** [comment]
It should be already fixed now.

**Comment by celestian** [comment]
And I am just observer, I don't have permission to merge it. :-(

**Comment by redoper1** [comment]
Yeah I know, it was only for information. 🙂

File: locales/cs.lua [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Locales ['cs'] = {
+
```
Comment (ElPumpo) [inline locales/cs.lua]: Remove blank line

File: locales/cs.lua [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  ['buy_license'] = 'koupit licenci?',
+  ['yes'] = 'ano',
+  ['no'] = 'ne',
+  ['buy'] = 'koupili jste %s',
```
Comment (ElPumpo) [inline locales/cs.lua]: I recently updated the code to use colors, and how much you bought the weapon for

File: locales/cs.lua [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  ['not_enough_black'] = 'nemáte dostatek špinavých peněz',
+  ['not_enough'] = 'nemáte dostatek peněz',
```
Comment (ElPumpo) [inline locales/cs.lua]: Under the line `not_enough` I've added a new one
```
['already_owned'] = 'you already own this weapon!',
```

translate this aswell

**Comment by ElPumpo** [comment]
Why did you add a SQL file with one line modified? Get it removed.

**Comment by redoper1** [comment]
> Why did you add a SQL file with one line modified? Get it removed.

I translated there a license for weapons.

**Comment by ElPumpo** [comment]
I know, it's one line - you'll have to remove the file if you want this PR merged.

**Comment by celestian** [comment]
@ElPumpo Could I ask you, what's the issue here? If @redoper1 remove the cs_esx_weashops.sql file, the Czech translation is not complete. So, I don't understand what's the issue here. Are you worry about size of file? Or what's the problem here?

**Comment by ElPumpo** [comment]
As I said it's one line changed, that's why I don't want multiple sql files for one line difference. Notice that no other translation has a sql file either, simply because it's bloaty.

**Comment by redoper1** [comment]
That's really stupid, good luck with your framework guys. It's getting worse and worse in these matters.

**Pull request closed without merging**